libs/blocks/marketo-config/marketo-config.js

```diff
@@ -267,16 +359,48 @@ const ConfiguratorWrapper = ({ title, link }) => {
 
   return html`
   <${ConfiguratorProvider} defaultState=${defaults} lsKey=${lsKey}>
-    <${Configurator} title=${title} blockClass=${blockClass} panelsData=${data} lsKey=${lsKey} />
+    <${Configurator} title=${title} panelsData=${data} lsKey=${lsKey} />
   </${ConfiguratorProvider}>
   `;
 };
 
+function createDataBlock(blockClass, data) {
+  const url = getDataUrl(data);
+  const link = createTag('a', { href: url }, url);
+  const row = createTag('div', null, link);
+  const block = createTag('div', { class: blockClass }, row);
+
+  return block;
+}
+
 export default async function init(el) {
   const children = Array.from(el.querySelectorAll(':scope > div'));
   const title = children[0].textContent.trim();
   const linkElement = children[1].querySelector('a[href$="json"]');
   const link = linkElement?.href;
+  // If the block is loaded in an iframe, replace it with the block itself
```

Comment:
did you want to keep this comment in?

Reply:
Yes, it's unusual for a block to act differently when in an iframe, but that's how I had to load it to get a consistent form experience. I could work around the reinitialization issues but isolating the form is a better longterm solution.